### PR TITLE
Add installer for symfony

### PR DESCRIPTION
**Pull request by apfelbox**

### CODE REVIEW

File: bin/janus
Comment (keichinger) [inline bin/janus:1]: CS: No more leading slashes?

File: bin/janus [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+
+// region Commands
+function initializeSymfony (SymfonyStyle $io) : int
+{
+	$io->writeln("• Copying config files to the project...");
+	copyFilesIntoProject($io, "symfony");
+
+	$io->writeln("• Updating <fg=yellow>composer.json</>...");
+	addToProjectComposerJson([
```
Comment (keichinger) [inline bin/janus:66]: How should we handle our Private Packagist configs? Do people still have to manually copy it into their projects?

File: bin/janus
Comment (apfelbox) [reply bin/janus:1]: The file has no namespace, so leading slashes make no difference.

File: bin/janus (hunk 1 above)
Comment (apfelbox) [reply bin/janus:66]: What does that have to do with the changes here?

File: bin/janus
Comment (keichinger) [reply bin/janus:1]: We'd require the configuration of the `repositories` key. Something that we could easily automate.

Comment (apfelbox) [reply bin/janus:1]: (the comment is in the wrong thread)
But that has nothing to do with "code style", and even less in our public open source library.

But that is already covered in our starter, so new projects should never need to add that.

**Review by keichinger — APPROVED** [review]

**Pull request merged**